css/utility.css

```diff
@@ -0,0 +1,118 @@
+.border-box {
+  box-sizing: border-box;
```

Comment:
you applied this property to universal selector in your reset css, is it affecting the flow of document in any way?